cache/redis_internal_test.go

```diff
@@ -45,7 +45,29 @@ func TestNewRedis(t *testing.T) {
 	c := v.(*redisCache)
 
 	assert.NoError(t, err)
-	assert.Equal(t, 12, c.client.Options().PoolSize)
+	clusterClient, isCluster := c.client.(*redis.ClusterClient)
+	if isCluster {
+		assert.Equal(t, 12, clusterClient.Options().PoolSize)
+	} else {
+		client, ok := c.client.(*redis.Client)
```

Comment:
Rather assert this is a non-clustered client, like we expect it to be.

Reply:
ok